packages/builders/src/components/v2/Container.ts

```diff
@@ -34,13 +36,31 @@ export interface ContainerBuilderData extends Partial<Omit<APIContainerComponent
 	components: ContainerComponentBuilders[];
 }
 
+/**
+ * A builder that creates API-compatible JSON data for containers.
+ */
 export class ContainerBuilder extends ComponentBuilder<APIContainerComponent> {
+	/**
+	 * @internal
+	 */
 	protected readonly data: ContainerBuilderData;
 
-	public constructor({ components = [], ...rest }: Partial<APIContainerComponent> = {}) {
+	/**
+	 * Gets the components within this container.
+	 */
+	public get components(): readonly ContainerComponentBuilders[] {
+		return this.data.components;
+	}
+
+	/**
+	 * Creates a new container builder.
+	 *
+	 * @param data - The API data to create the container with
```

Comment:
This would document the wrong parameters, as `data` now only refers to the rest without`components`. For documentation purposes it would be better to name the full parameter `data` and destructure first thing in the body. Similar to how it was in other constructors before this change.